rollup/rollup-cdn-common.js

```diff
@@ -0,0 +1,79 @@
+import path from 'path';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import resolve from '@rollup/plugin-node-resolve';
+import {
+  babelPlugin,
+  clearPlugin,
+  getMainFiles,
+  importMetaUrlAssetsPlugin,
+  manualChunkOptions,
+  minifyStylesheet,
+  terserPlugin,
+  treeshakeOptions,
+  visualizerPlugin,
+} from './rollup-common.js';
+import { depsManifestPlugin } from './rollup-plugin-deps-manifest.js';
+import { stylesAssetsPlugin } from './rollup-plugin-styles-assets.js';
+
+const inputFilesPairs = getMainFiles('src').map((file) => {
```

Comment:
suggestion: at this point, I think `src` should only be defined once in `rollup-common.js` for `getMainFiles()` and `getAllSourceFiles()` so they don't take any params and users of the function don't need to maintain the string in multiple places.